lib/common.js

```diff
@@ -81,7 +91,12 @@ class FileHasher extends stream.Writable {
     }
 
     _write(chunk, encoding, next) {
-        this._hasher.update(chunk);
+        try {
```

Comment:
don't think this is necessary in this case (the other 2 cases yes but not here) because the stream is not set to object mode and node does a damn good job of validating that its a string that's passed into the write method. I haven't been able to get anything into `hasher.update` that could blow up. I can only get strings in there and all strings are valid args to `.update`

Reply:
Might not be, but node docs says to always use the callback, so doesn't hurt, does it?

https://nodejs.org/api/stream.html#stream_errors_while_writing